api/api/accounts/test_list_userroles.py

```diff
@@ -0,0 +1,58 @@
+from datetime import date
+
+import pytest
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from api.clubs.models import Club
+from api.roles.models import Roles
+
+User = get_user_model()
+
+
+class ListUserRolesTest(APITestCase):
```

Comment:
Prefer function based tests. `pytest` is more friendly for function based tests.

Refer: https://github.com/BitByte-TPC/gymkhana/blob/master/api/api/events/test_list_create_events.py